labelbox/schema/slice.py

```diff
@@ -0,0 +1,64 @@
+from typing import List
+
+from labelbox.orm.db_object import DbObject
+from labelbox.orm.model import Field
+
+
+class Slice(DbObject):
+    """
+    A Slice is a saved set of filters (saved query) used to find data rows matching a certain criteria
+
+    Attributes:
+        name (datetime)
+        description (datetime)
+        created_at (datetime)
+        updated_at (datetime)
+        filter (json)
+    """
+    name = Field.String("name")
+    description = Field.String("description")
+    created_at = Field.DateTime("created_at")
+    updated_at = Field.DateTime("updated_at")
+    filter = Field.Json("filter")
```

Comment:
Are all of these guaranteed to exist on the ModelSlice? I assume they will behave the exact same?

Reply:
I suggest to keep both catalog and model slices in the same Mysql table, so yes, these props would exists for both types of slices.